README.md

```diff
@@ -29,102 +39,26 @@ This project has 2 parts: the Docker environment and a set of tools for theme de
 ```
 # copy the files
 git clone https://github.com/visiblevc/wordpress-starter.git
-rm -rf .git Dockerfile run.sh README.md CHANGELOG.md ISSUE_TEMPLATE.md
+
+# navigate to example directory
+cd wordpress-starter/example
 
 # start the website at localhost:8080
-docker-compose up
+docker-compose up -d && docker-compose logs -f wordpress
```

Comment:
I haven't run a wordpress site in a while but I think `docker-compose up` did output the logs for me. It is arguably more verbose though since it logs all services and not just `wordpress`.